### DISCUSSION TITLE
Running probes on local model

### DISCUSSION THREAD

**Discussion by Walfar** [NVIDIA/garak#283] [Q&A]
Hello everyone,

I love this library, great job! I was wondering if it was possible to run the probes, e.g art, on a local model, hence that is neither available on huggingface nor openai.

Best,
W

**Comment by leondz** [comment]
Hi Walfar,

Sorry for the delayed response. It's possible, but we only have a few interfaces for it for now: function, huggingface (model or pipeline), or ggml. What kind of interface/data structure does the local model in mind have?
